charts/linkerd-control-plane/Chart.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
 - name: partials
   version: 0.1.0
   repository: file://../partials
-version: 1.15.0
+version: 1.15.1
```

Comment:
This looks good but we tend to bump chart versions on releases as opposed to individual changes. We should omit this change.

Reply:
Shoud I do something or will you ?

Reply:
Would you mind taking care of both changes? :D It makes it a bit easier that way.

Reply:
ok no problem